utils/posts.js

```diff
@@ -43,7 +43,15 @@ export function getPostMetadata(id) {
     };
 
     const title = post.meta.title;
-    const seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, '-');
+    var seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, '-');
+
+    //Converts all dashes to spaces (whitespace)
+    seoTitle = seoTitle.replace(/\-/g, ' ');
+    //Removes whitespace from the ends
+    seoTitle = seoTitle.trim();
+    // Converts all spaces back to dashes
+    seoTitle = seoTitle.replace(/ /g, '-');
```

Comment:
This can be shortened even more.

The first replacement should be to spaces, and then trimmed, and then convert the spaces back to dashes.

It can all be chained (done inline), as opposed to reassigning the variable multiple times.

Reply:
Here's a screenshot illustrating the potential solution

(However, instead of using `.toLocaleLowerCase()`, just use `.toLowerCase()`)

![image](https://user-images.githubusercontent.com/422501/113661708-72f85600-965b-11eb-9566-af3bd6b00527.png)

Reply:
Actually, `.toLocaleLowerCase()` is even better, because it can handle even more characters:

`'Épicenter'.toLocaleLowerCase()` -> `"épicenter"`

Reply:
Nevermind, `.toLowerCase()` also does that.